Plugin/src/SofaPython3/PythonEnvironment.cpp

```diff
@@ -233,25 +235,16 @@ void PythonEnvironment::Init()
     // Lastly, we (try to) add modules from the root of SOFA
     addPythonModulePathsFromDirectory( Utils::getSofaPathPrefix() );
 
-    try
-    {
-        py::module::import("SofaRuntime");
-    }
-    catch (pybind11::error_already_set)
-    {
-        msg_error("SofaPython3") << "Could not import SofaRuntime module, initializing python3 for SOFA is not possible";
-        return;
-    }
     getStaticData()->m_sofamodule = py::module::import("Sofa");
-
+    PyRun_SimpleString("import SofaRuntime");
```

Comment:
Why this replacement?

Reply:
Because as far as I can see it is not the same and the other was not working as expected.
The difference is that the other is importing something in a standalone context, accessible from c++ and not importing a module it into the current interpreter context.

And this is how other line of code in this file are doing for the same behavior eg: 
```cpp
PyRun_SimpleString("from Sofa.livecoding import onReimpAFile");
```